modules/hostap/Kconfig

```diff
@@ -412,6 +412,11 @@ config RRM
 	default y
 	depends on WIFI_NM_WPA_SUPPLICANT_RRM
 
+config RRM_MSR_REPORT_LEN
```

Comment:
is there a hostap PR where this is used?

Reply:
yes, https://github.com/zephyrproject-rtos/hostap/pull/24
Hi @krish2718  please review again

Reply:
Hi @jukkar @krish2718  please review again